copter/source/docs/reference-frames-hexsoon-td650.rst

```diff
@@ -22,31 +23,73 @@ Box Contents
 - 8000mAh to 12000mAh 6S battery
 - RC transmitter with at least 6 channels
+- (Optional) four `5cm (2inch) C-clamps <https://www.amazon.com/C-Clamp-Industrial-Strength-Woodworking-Building/dp/B0BLC44XHT>`__ to ease assembly
+- (Optional) `solder sleeves <https://www.amazon.com/TICONN-Connectors-Waterproof-Insulated-Electrical/dp/B07GDDKJ1D>`__ to shorten ESC wires
+- (Optional) 2cm x 50cm velcro battery straps to replace very short straps provided
 
 Connection and Setup
 --------------------
 
-.. image:: ../images/refrence-frames-hexsoon-edu650-assembly.jpg
-    :target: ../_images/refrence-frames-hexsoon-edu650-assembly.jpg
+.. image:: ../images/reference-frames-hexsoon-edu650-assembly.jpg
+    :target: ../_images/reference-frames-hexsoon-edu650-assembly.jpg
 
 Connect the four ESC wires to the back of the autopilot as shown in the :ref:`QuadX configuration <connect-escs-and-motors>`
 
 Parameter file: `hexsoon-td650.param <https://github.com/ArduPilot/ardupilot/blob/master/Tools/Frame_params/hexsoon-td650.param>`__
 
 This parameter file can also be loaded using the Mission Planner's Config/Tuning >> Full Parameter Tree page by selecting "hexsoon-td650" from the drop down on the middle right and then push the "Load Presaved" button.
 
+LED connection and configuration
+--------------------------------
+
+The LED cable from each arm should be connected to the PDB's LED ports (highlighted in green below) labelled "1" to "4" (the order does not matter).
+The adjacent 6-pin port (highlighted in purple below) should be connected to two of the autopilot's PWM output pins (e.g. AUX OUT 5 and 6)
+
+  .. image:: ../images/reference-frames-hexsoon-edu650-assembly-LEDs.png
+      :target: ../_images/reference-frames-hexsoon-edu650-assembly-LEDs.png
+      :width: 450px
+
+Then set the following parameters on the autopilot, assuming the autopilot's AUX OUT 5 and 6 pins have been connected as shown above
+
+- :ref:`SERVO13_FUNCTION <SERVO13_FUNCTION>` = 129 (ProfiLED1)
+- :ref:`SERVO14_FUNCTION <SERVO14_FUNCTION>` = 132 (ProfiLEDClock)
+- :ref:`NTF_LED_LEN <NTF_LED_LEN>` = 8
+- Check the :ref:`NTF_LED_TYPES <NTF_LED_TYPES>` bit for "ProfiLED"
+
+  .. image:: ../images/reference-frames-hexsoon-edu650-params1.png
+      :target: ../_images/reference-frames-hexsoon-edu650-params1.png
+
+Build issues and advice
+-----------------------
+
+Below is a list of issues and recommendations
+
+- Assembly is difficult because both upper and lower plates are required to hold the legs in place.  Temporarily use four `5cm (2inch) C-clamps <https://www.amazon.com/C-Clamp-Industrial-Strength-Woodworking-Building/dp/B0BLC44XHT>`__ to hold the lower plate and legs together.
+- The ESC wires are too long resulting in messy wiring.  These can be shortened using `solder sleeves <https://www.amazon.com/TICONN-Connectors-Waterproof-Insulated-Electrical/dp/B07GDDKJ1D>`__
```

Comment:
Where should we put the ESC's? One diagram shows them in the arms, another place zip tied them to the bottom plate.

Reply:
Txs.  In my vehicle i've mounted the PDB to the underside of the top plate.  I've shortened the power wires to the ESC and attached them to the underside of the top plate as well.  In my build at least, the ESCs would not fit within the arms.

I think it would be add some images of suggested mounting positions for all parts but I think we should first merge this as-is and then iterate with future improvements.